lib/rules.rb

```diff
@@ -14,19 +17,19 @@ class Rules
     [2, 4, 6]
 ]
 
-  def game_over?(board)
-    winning_combination?(board) || check_for_draw?(board)
+  def game_over?
+    winning_combination? || check_for_draw?
   end
 
-  def winning_combination?(board)
+  def winning_combination?
     WINNING_COMBOS.any? do |x, y, z|
-      board.position_taken?(x) &&
-      board.squares[x] == board.squares[y] &&
-      board.squares[x] == board.squares[z]
+      @board.position_taken?(x) &&
+      @board.squares[x] == @board.squares[y] &&
+      @board.squares[x] == @board.squares[z]
     end
   end
 
-  def check_for_draw?(board)
-    winning_combination?(board) == false && board.full? == true
+  def check_for_draw?
+    winning_combination? == false && @board.full? == true
   end
```

Comment:
Since this method just returns a boolean, you could name it `draw?` (I think `check` is redundant - this is subjective though. It's not a bad name!)

More importantly, you don't need to explicitly compare your booleans to `true`/`false` - that *is* redundant. The body of this method can be
`!winning_combination? && @board.full?`